src/module-2/components/index.html

```diff
@@ -0,0 +1,340 @@
+</head>
+<body>
+<header>
+  <div class="logo">
+    <img alt="Logo" src="images/logo.svg">
+    <p>Online Store</p>
+  </div>
+  <nav>
+    <a href=""><span>&#8962;</span></a>
+    <img src="images/arrows.png">
+    <a href="">eCommerce</a>
+    <img src="images/arrows.png">
+    <a href="">Electronics</a>
+  </nav>
+</header>
+<div class="filters">
+  <div class="caption">
+    <span>Filters</span>
+    <button><img src="images/arrows.png" alt="arrows"></button>
+  </div>
+
+  <div class="filter-card">
+    <span class="filters-caption">Multi Range</span>
+    <div>
+      <p><input name="range" type="radio">$10</p>
+      <p><input name="range" type="radio">$10-$100</p>
+      <p><input name="range" type="radio">$100-$500</p>
+      <p><input name="range" type="radio">$500</p>
+      <p><input name="range" type="radio">All</p>
+      <div class="divider"></div>
+      <span class="filters-caption">Category</span>
+      <p><input type="checkbox">Cell Phones</p>
+      <p><input type="checkbox">Computers & Tablets</p>
+      <p><input type="checkbox">Cell Phones Accessories</p>
+      <p><input type="checkbox">Appliances</p>
+      <p><input type="checkbox">Audio</p>
+      <p><input type="checkbox">iPhone Accessories</p>
+      <p><input type="checkbox">Cameras & Camcorders</p>
+      <p><input type="checkbox">iPhone Cases & Clips</p>
+      <p><input type="checkbox">TV & Home Theater</p>
+      <p><input type="checkbox">Small Kitchen Appliances</p>
+      <div class="divider"></div>
+      <span class="filters-caption">Brand</span>
+      <p><input type="checkbox">Insigni</p>
+      <p><input type="checkbox">Samsung</p>
+      <p><input type="checkbox">Apple</p>
+      <p><input type="checkbox">HP</p>
+      <p><input type="checkbox">Sony</p>
+      <p><input type="checkbox">Metra</p>
+      <p><input type="checkbox">Dyne</p>
+      <p><input type="checkbox">LG</p>
+      <p><input type="checkbox">Canon</p>
+      <p><input type="checkbox">Speck</p>
+    </div>
+  </div>
+  <button>clear all filters</button>
+</div>
+
+<div class="results">
+  <div class="caption">
+    <span>7,618 results found</span>
+    <button><img src="images/wishlist.svg"></button>
+  </div>
+  <div class="under-caption">
+    <div class="search">
+      <input placeholder="Search" type="search">
+      <button type="submit"><i class="fa fa-search"></i></button>
+    </div>
+    <main>
+      <section class="card">
+        <figure>
+          <img class="product" src="images/product.png" alt="Apple Watch">
+        </figure>
+
+        <div class="rate_cost">
+          <div class="rate">
+            <span>3.4</span>
+            <img class="icon" src="images/rate-star.svg" alt="Rate Star">
+          </div>
+
+          <div class="cost">
+            <span>399$</span>
+          </div>
+        </div>
+
+        <article class="description">
+          <span>Apple Watch Series 4 GPS</span>
+          <p>Redesigned from scratch and completely revised.</p>
```

Comment:
minor: consider to use optimal font-size for responsive in the next modules. For more info, please refer to [mobile-friendly design mistakes](https://developers.google.com/search/mobile-sites/mobile-seo/common-mistakes)